test/e2e/secrets_test.go

```diff
@@ -116,7 +116,7 @@ func (tc *testContext) waitForNewMachineNodes() error {
 	for _, newNode := range gc.machineNodes {
 		for _, oldNode := range oldNodes {
 			if newNode.GetName() == oldNode {
-				return errors.Wrapf(err, "node %s is not a new Node", oldNode)
+				return fmt.Errorf("node %s is not a new Node: %w", oldNode, err)
```

Comment:
I don't think this error needs to be wrapped. `err` will always hold `nil` value here (if it's non-nil it is returned before entering the loop